anitya/default_config.py

```diff
@@ -48,3 +50,16 @@
 EMAIL_ERRORS = False
 
 BLACKLISTED_USERS = []
+
+# The location of the client_secrets.json file used for API authentication
+OIDC_CLIENT_SECRETS = os.path.join(
+    os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+    'client_secrets.json'
+)
+
+# Force the application to require HTTPS to save the cookie. This should only
+# be `False` in a development environment running on the local host!
+OIDC_ID_TOKEN_COOKIE_SECURE = True
+OIDC_REQUIRE_VERIFIED_EMAIL = True
+OIDC_OPENID_REALM = 'http://localhost:5000/oidc_callback'
+OIDC_SCOPES = ['openid', 'email', 'profile', 'fedora']
```

Comment:
Please update this to use the groups and cla scopes registered on the Authentication documentation page